doc/rados/configuration/mon-config-ref.rst

```diff
@@ -357,6 +357,17 @@ by setting it in the ``[mon]`` section of the configuration file.
 :Default: ``True``
 
 
+``mon warn osd usage percent``
```

Comment:
this option was replaced by "mon_warn_osd_usage_min_max_delta", which was in turn removed  in 17c1a8f74a4d4985e936cbff4276e86eae619d3c.

Reply:
okay, so both the options seems like outdated, should I close this pull request or is there any option still not well documented that needs to be added?